blazor-features/BlazorWebAssemblyCustomEvents/BlazorWebAssemblyCustomEvents/CustomEvents/EventHandlers.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.AspNetCore.Components;
+using Microsoft.AspNetCore.Components.Forms;
+
+namespace BlazorWebAssemblyCustomEvents.CustomEvents
+{
+    [EventHandler("ondoubleclick", typeof(DoubleClickEventArgs), enableStopPropagation: true, enablePreventDefault: true)]
+    public static class EventHandlers
+    {
+        // purposefully empty
+    }
+
+    public class DoubleClickEventArgs : EventArgs
+    {
+        public DateTime EventTimestamp { get; set; }
+        public bool isDoubleClick { get; set; }

```

Comment:
Sorry I missed that on the first review, but properties inside classes should always follow the Pascal-Case naming convention. Based on that, the `isDoubleClick` property should start with the uppercase -> `IsDoubleClick`